src/compiler/binder.ts

```diff
@@ -258,7 +259,7 @@ namespace ts {
                 case SyntaxKind.ExportDeclaration:
                     return "__export";
                 case SyntaxKind.ExportAssignment:
-                    return (<ExportAssignment>node).isExportEquals ? "export=" : "default";
+                    return ((<ExportAssignment>node).isExportEquals ? "export=" : "default");

```

Comment:
looks like some extra parens got left in after their associated casts were removed.